mmv1/third_party/terraform/website/docs/d/vpc_access_connector.html.markdown

```diff
@@ -0,0 +1,65 @@
+---
+subcategory: "Serverless VPC Access"
+page_title: "Google: google_vpc_access_connector"
+description: |-
+  Get a Serverless VPC Access connector.
+---
+
+# google\_vpc\_access\_connector
+
+Get a Serverless VPC Access connector.
+
+To get more information about Connector, see:
+
+* [API documentation](https://cloud.google.com/vpc/docs/reference/vpcaccess/rest/v1/projects.locations.connectors)
+* How-to Guides
+    * [Configuring Serverless VPC Access](https://cloud.google.com/vpc/docs/configure-serverless-vpc-access)
+
+## Example Usage
+
+```hcl
+data "google_vpc_access_connector" "sample" {
+  name = "vpc-con"
+}
+
+resource "google_vpc_access_connector" "connector" {
+  name          = "vpc-con"
+  ip_cidr_range = "10.8.0.0/28"
+  network       = "default"
+  region        = "us-central1"
+}
+```
+
+## Argument Reference
```

Comment:
We'll want to format the argument and attribute reference similarly to https://registry.terraform.io/providers/hashicorp/google/latest/docs/data-sources/compute_backend_bucket, where we list the fields required to specify the datasource' resource, and link to the resource for the attributes. We're not realistically gonna keep the resource and datasource docs in sync, otherwise!

Filed https://github.com/hashicorp/terraform-provider-google/issues/12559 to be more explicit in our docs.